lua/entities/starfall_processor/cl_init.lua

```diff
@@ -185,3 +185,75 @@ net.Receive("starfall_processor_used", function(len)
 	end
 end)
 
+local function getId( arg )
```

Comment:
This function could be called getPlayerFromSteamID, and return a player or nil given the concommand argument. That would clean up duplicate code in those concommands.

Reply:
It might also be worth trying and seeing if player.GetBySteamID works with either steamid format on its own. Maybe it can do the conversion itself.

Reply:
We can look into that, sure

Reply:
> It might also be worth trying and seeing if player.GetBySteamID works with either steamid format on its own. Maybe it can do the conversion itself.

`player.GetBySteamID` uses `Player:SteamID` which is distinct from `Player:SteamID64`

Reply:
Also, what is this `util.SteamIDFrom64` anyway

Reply:
![image](https://github.com/thegrb93/StarfallEx/assets/45838657/8fbde2d3-80b9-4523-81e1-fa8384c444cd)

Reply:
We've got two options here. We can use `player.GetBySteamID` and `player.GetBySteamID64`, or we can use `util.SteamIDFrom64` if `tonumber( id )` passes, and call `player.GetBySteamID`.

Reply:
Okay, I went with the latter option ^

Needs testing, should apply all changes requested